src/app/public/modules/date-range-picker/types/date-range-calculator-id.ts

```diff
@@ -1,24 +1,140 @@
+/**
+ * `SkyDateRangeCalculatorId` values specify calculator objects that return
+ * two `Date` objects to represent date ranges. The `SkyDateRangeCalculatorId` values
+ * are used to populate the options in the date range picker's dropdown.
+ * SKY UX uses the `SkyDateRangeService` to create calculators and configures each one
+ * with a `validate` function to confirm that the dates users select are compatible. For example,
+ * `validate` functions ensure that start dates are before end dates. SKY UX also
+ * configures calculators to call a `getValue` function after the `validate` function and return
+ * a range of two `Date` objects.
+ */
```

Comment:
☝️ FYI - these inline code blocks don't format correctly. I've filed an issue for the bug: https://github.com/blackbaud/skyux-docs-tools/issues/69

Reply:
I think I commented on this as well in my feedback. Since we have filed an issue in docs-tools, I think we can move forward with these code snippets as-is for now. I'm resolving the conversation, but let me know if I'm wrong and we need to unresolve it and hold off on this PR until the docs-tools issue is addressed.